{"name": "Проверка ошибки при попытке удаления курьера без указания id", "status": "passed", "steps": [{"name": "Удаление курьера", "status": "passed", "parameters": [{"name": "courier_id", "value": "''"}], "start": 1744280278086, "stop": 1744280278152}], "start": 1744280278086, "stop": 1744280278152, "uuid": "27daa47a-250f-4478-b7fb-a4f12d689da0", "historyId": "d3df7bfe33bd8b710b6e4ecfeb6ee313", "testCaseId": "d3df7bfe33bd8b710b6e4ecfeb6ee313", "fullName": "tests.tests_couriers.test_courier_delete.TestCourierDelete#test_courier_delete_error_with_empty_id", "labels": [{"name": "parentSuite", "value": "tests.tests_couriers"}, {"name": "suite", "value": "test_courier_delete"}, {"name": "subSuite", "value": "TestCourierDelete"}, {"name": "host", "value": "Evgeny"}, {"name": "thread", "value": "16148-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.tests_couriers.test_courier_delete"}]}